src/arviz_plots/plots/combine.py

```diff
@@ -0,0 +1,95 @@
+import re
+from importlib import import_module
+
+import xarray as xr
+from arviz_base import rcParams
+
+from arviz_plots import PlotCollection
+from arviz_plots.plots.utils import process_group_variables_coords
+
+def get_valid_arg(key, value, backend):
+    plot_backend = import_module(f"arviz_plots.backend.{backend}")
+    key_matcher = "color" if key in {"facecolor", "edgecolor"} else key
+    if isinstance(value, str) and re.match(key_matcher +"_[0-9]+", value):
+        _, index = value.rsplit("_", 1)
+        index = int(index)
+        return plot_backend.get_default_aes(key, index+1)[index]
+    return value
+
+
+def backendize_kwargs(kwargs, backend):
+    return {key: get_valid_arg(key, value, backend) for key, value in kwargs.items() if key != "function"}
+
+def render(da, target, backend, **kwargs):
+    plot_backend = import_module(f"arviz_plots.backend.{backend}")
+    plot_kwargs = da.item()
+    plot_fun_name = plot_kwargs["function"]
+    plot_kwargs = backendize_kwargs(plot_kwargs, backend)
+    return getattr(plot_backend, plot_fun_name)(target=target, **{**plot_kwargs, **kwargs})
+
+def combine_plots(
+    dt,
+    plots=None,
+    expanded_dim="column",
+    expanded_coord_names=None,
+    group="posterior",
+    var_names=None,
+    filter_vars=None,
+    coords=None,
+    backend="matplotlib",
+):
+    if expanded_coord_names is None:
+        expanded_coord_names = [
+            getattr(elem[0], "__name__") + f"_{idx:02d}" for idx, elem in enumerate(plots)
+        ]
+    sample_dims = rcParams["data.sample_dims"]
+    column_length = len(expanded_coord_names)
+
+    distribution = process_group_variables_coords(
+        dt, group=group, var_names=var_names, filter_vars=filter_vars, coords=coords
+    )
+    facet_dims = ["__variable__"] + ([] if "predictive" in group else [dim for dim in distribution.dims if dim not in sample_dims])
```

Comment:
This is quite the assumption, we might want to make this an argument of the function with that as default